CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift

```diff
@@ -0,0 +1,321 @@
+//
+//  Camera2ViewController.swift
```

Comment:
[Clarification] Will this end up being renamed to `CameraViewController` to replace the old one?

Reply:
Not sure, I left the old version as agreed during discussion. We can always replace/rename it.

Reply:
I think I missed that discussion. I will ask for a reminder in our daily today 🙈.